pages/1.12/storage/partner integration/index.md

```diff
@@ -0,0 +1,105 @@
+# Partner Integrations
```

Comment:
WHERE IS THE METADATA?

Reply:
What does this mean? Can you please explain?

Reply:
Couldn't find an easy way to change the folder name. If you can do it, please do.